src/typings/generic.ts

```diff
@@ -7,6 +7,7 @@ type ObjectKeys<T> = T extends object
   ? string[]
   : never;
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
How come we need this here?

Reply:
Another piece of code I don't understand how it works. This file exports nothing but is still needed or we get compilation errors.

Reply:
See the link at the top of the file

Reply:
yeah i think this is fine